test/integration/MdgWcoDeclarationConnectorSpec.scala

```diff
@@ -44,6 +45,8 @@ class MdgWcoDeclarationConnectorSpec extends IntegrationTestSpec with GuiceOneAp
 
   private val incomingBearerToken = "some_client's_bearer_token"
   private val incomingAuthToken = s"Bearer $incomingBearerToken"
+  private val numberOfCallsToTriggerStateChange = 5
+  private val unavailablePeriodDurationInMillis = 1500
```

Comment:
I think we should reduce this time of 1500ms, too long for a test

Reply:
reduced to 1000